Jackal/Jackal Shared/Playing Field/Shared/GraphConnectors/IceNodeConnector.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  IceNodeConnector.swift
+//  Jackal iOS
+//
+//  Created by Alexander Savonin on 20/03/2018.
+//  Copyright © 2018 Boring Company. All rights reserved.
+//
+
+class IceNodeConnector: NodeConnectorDescribing {
```

Comment:
Really, it seems what its better to do some custom class for SubNodes and bring back struct, @tapoton What do U think?